Assets/Scripts/KeyBindings/StoreKeyBindings.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+using Newtonsoft.Json;
+
+
+public static class StoreKeyBindings
+{
+    private const string custom_path = "keybindings_custom.json";
+    private const string default_path = "keybindings_default.json";
+
+    static StoreKeyBindings()
+    {
+      
+    }
+        
+    public static void SaveKeyBindings()
+    {
+        Dictionary<String, KeyCode> bindings = KeyBindings.Get();
+
+        var content = JsonConvert.SerializeObject(bindings);
+        File.Delete(custom_path);
+        File.WriteAllText(custom_path, content);
+    }
+
+    public static bool SetDefaultBindings()
```

Comment:
What should this function do?
Should it transfer the constant default bindings into the custom bindings? But what is the difference to RestoreDefaults then?
Why is it called SetDefaultBindings and prematurely exits, when the custom config does not exist?

Reply:
changed name to SetCustomBindings